src/be/app.py

```diff
@@ -28,16 +32,52 @@ def static(path: str):
         path=_fe_dir / path,
     )
 
-
 @app.get("/api/v1/status")
 def status():
+    #Use the `bulb` variable to get the properties of the LED strip, such as its name, power status, brightness, and color. Return these properties as an instance of the `Status` class.
+    properties = bulb.get_properties()
+    bulb_name_variable = properties['name'] or "Unknown"
+    power_status = properties['power']
+    brightness = properties['bright']
+    colour = properties['rgb']
+    if bulb_name_variable is None:
+        bulb_name_variable = 'Unknown'
+    print(f"Bulb name: {bulb_name_variable}, Power status: {power_status}, Brightness: {brightness}, Colour: {colour}")
     return Status(
-        bulb_name="TODO",
+        bulb_name= bulb_name_variable,
+        power_status='on',
```

Comment:
you can actually use the `power_status`, `brightness`, `colour` python variables here now, so that the status reflects the true state of the bulb

Reply:
Done